### ISSUE TITLE
Navbar menu isn't accessible at high zoom levels

### ISSUE THREAD

**Issue by codyduong** [codyduong/codyduong.dev#8]
The overflow-y value is disabled, meaning if you scroll in too much, you cannot access lower links in the menu, re-enable scroll/allow overflow-y
